ui/views_test.py

```diff
@@ -105,6 +105,7 @@ def test_video_detail(logged_in_client, settings):
     assert js_settings_json == {
         'editable': True,
         "gaTrackingID": settings.GA_TRACKING_ID,
+        "ga_dimension_camera": settings.GA_DIMENSION_CAMERA,
```

Comment:
Can you set `settings.GA_TRACKING_ID` and `settings.GA_DIMENSION_CAMERA` at the top of the test so that it doesn't use the default of `""`?

Reply:
✅